interfaces/external-processor-renesas/Readme.md

```diff
@@ -0,0 +1,166 @@
+
+# Booting the DA14531 through a R7FA2E1 from RENESAS (EK-RA2E1)
+
+---
+## Example description
+
+The DA14531 has an integrated ARM Cortex M0+ and can be used for programming and controlling the SoC. 
+Due to the small form factor the ARM Cortex has its limitations. A solution for the limitations of the integrated microprocessor can be to use an external microprocessor. 
+To control the DA14531 with this microprocessor some code is still needed on the DA14531. 
+Fortunately, the DA14531 can execute code from RAM that can be loaded in during its boot sequence. 
+This way a microcontroller can load a program into the DA14531.
+
+The goal of this example is to show how to load a program into the RAM of the DA14531 via a R7FA2E1 (EK-RA2E1) microcontroller. 
+This example shows the flow of the code and how it can be configured. The program that is booted on the DA14531 in this example is the Codeless or proximity reporter depending on the user selection in **`da14531_boot_config.h`**. 
```

Comment:
```suggestion
This example shows the flow of the code and how it can be configured. The program that is booted on the DA14531 in this example is Codeless or proximity reporter depending on the user selection in **`da14531_boot_config.h`**. 
```